### PR TITLE
translate WIP - feedback request

### PR DESCRIPTION
**Pull request by ctava**
@shenwei356 kindly review this PR for #28 and let me know what you think.

### CODE REVIEW

File: seqkit/cmd/common.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ Note:
 		// lineWidth := config.LineWidth
 		outFile := config.OutFile
 		quiet := config.Quiet
-		seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
+		//seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
```
Comment (shenwei356) [inline seqkit/cmd/common.go]: Why removing?

File: seqkit/cmd/sort.go [hunk 2]
```diff
@@ -63,14 +63,14 @@ and extract sequences by FASTA index.
...
-		seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
+		//seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
 		seq.ValidateSeq = false
 		fai.MapWholeFile = false
 		runtime.GOMAXPROCS(config.Threads)
 
 		files := getFileList(args)
 
-		inNaturalOrder := getFlagBool(cmd, "natural-order")
+		//inNaturalOrder := getFlagBool(cmd, "natural-order")
```
Comment (shenwei356) [inline seqkit/cmd/sort.go]: The reason?

File: seqkit/cmd/translate.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+				if err != nil {
+					if err == io.EOF {
+						break
+					}
+					checkError(err)
+					break
+				}
+
+				//mRNA to amino acid conversion - TODO move this to a better place
+				rnaToProteinMapping := map[string]string{
```
Comment (shenwei356) [inline seqkit/cmd/translate.go]: You know, there are [27 condon tables](https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=tgencodes#SG1). And translate methods better be integrated into github.com/shenwei356/bio/seq .

File: tests/mrna-drosopholia-DQ327736.fa [hunk 4]
```diff
@@ -0,0 +1,7 @@
+>DQ327736 1
```
Comment (shenwei356) [inline tests/mrna-drosopholia-DQ327736.fa]: better be real mRNA starting with `ATG`.

File: tests/test.sh [hunk 5]
```diff
@@ -395,3 +395,16 @@ fun(){
...
+#                       translate
+# ------------------------------------------------------------
+
+file=tests/mrna-drosopholia-DQ327736.fa
+
+fun(){
+    $app translate $file > t.translate.0
+}
+run translate fun
+assert_equal $(cat t.translate.0 | $app stat) "Protein"
```
Comment (shenwei356) [inline tests/test.sh]: obviously not equal.

File: seqkit/cmd/common.go (hunk 1 above)
Comment (ctava) [reply seqkit/cmd/common.go]: didn't compile. don't yet understand root cause. reproducible on your end?

File: seqkit/cmd/sort.go (hunk 2 above)
Comment (ctava) [reply seqkit/cmd/sort.go]: didn't compile. don't yet understand root cause. reproducible on your end?

File: seqkit/cmd/translate.go (hunk 3 above)
Comment (ctava) [reply seqkit/cmd/translate.go]: 27 👍 

agree - github.com/shenwei356/bio/seq

File: tests/mrna-drosopholia-DQ327736.fa (hunk 4 above)
Comment (ctava) [reply tests/mrna-drosopholia-DQ327736.fa]: could you kindly suggest a source of better test data?

File: tests/test.sh (hunk 5 above)
Comment (ctava) [reply tests/test.sh]: yep. ran out of time. will have to re-write / write tests..

**Pull request closed without merging**